application/features/Term.py

```
@@ -49,10 +51,12 @@ def __del__(self):
         super().__del__()
 
     def connect(self, *args, **kwargs):
```

Ruff

Missing return type annotation for public function `connect`

(ANN201)